ddl/db_test.go

```diff
@@ -7770,3 +7770,49 @@ func (s *testSerialDBSuite) TestAddGeneratedColumnAndInsert(c *C) {
 	tk.MustQuery("select * from t1 order by a").Check(testkit.Rows("4 5", "10 11"))
 	c.Assert(checkErr, IsNil)
 }
+
+func (s *testSerialDBSuite) TestIssue32079(c *C) {
```

Comment:
Could we use `TestDeleteOnlyForDropColumnWithIndexes` directly? We can add `update t set id = 4 where id=40;` to `sqls` in `TestDeleteOnlyForDropColumnWithIndexes` .